doonut-core/src/main/resources/db/migration/V20240501234456__calender_ddl.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS calender
+(
+    id                BIGINT AUTO_INCREMENT               NOT NULL COMMENT 'ID',
+    member_id         BIGINT                              NOT NULL COMMENT '멤버 ID',
+    calender_name     VARCHAR(100)                        NOT NULL COMMENT '캘린더 명',
+    total_count       INT                                 NOT NULL COMMENT '총 기록 횟수',
+    first_upload_date  TIMESTAMP                               NULL COMMENT '최초 업로드 날짜',
+    last_upload_date  TIMESTAMP                               NULL COMMENT '마지막 업로드 날짜',
```

Comment:
포맷터 한번 돌리면 좋을 것 같아요!  

```suggestion
    first_upload_date TIMESTAMP                           NULL COMMENT '최초 업로드 날짜',
    last_upload_date  TIMESTAMP                           NULL COMMENT '마지막 업로드 날짜',
```